.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+### Instructions
+
+The github issue tracker is for bug reports, feature requests, and API documentation requests. General questions about how to use the GATK, how to interpret the output, etc. should be asked on the [official support forum](http://gatkforums.broadinstitute.org/gatk).
+- Search the existing github issues to see if your issue (or something similar) has already been reported.  If the issue already exists, you may comment there to inquire about the progress.
+- Determine whether your issue is a **bug report**, a **feature request**, or a **documentation request** (for tool gatkdocs only -- for forum docs see [the support issues repo](https://github.com/broadinstitute/dsde-docs/issues))
```

Comment:
maybe tool gatk docs -> javadoc ?

Reply:
By the way, I guess that the repository is private - it might be a good idea to mention it, because I was trying to access and I was wondering why it didn't work. I can't submit an issue about forum docs, but I guess that I would not realize if there is an issue there.

Reply:
Oh. I didn't realize that. We should not link to a private repository.